Makefile

```
@@ -1,11 +1,19 @@
 CC=x86_64-w64-mingw32-gcc
 
-all: truckersmp-cli.exe
+all: truckersmp-cli.exe _truckersmp-cli
 
 truckersmp-cli.exe: truckersmp-cli.c
 	$(CC) $< -o $@
 
+_truckersmp-cli:
+	# genzshcomp <(./truckersmp-cli --help) > _truckermp-cli
+	if which genzshcomp &> /dev/null; then \
+		./truckersmp-cli --help > helptext; \
+		genzshcomp helptext > _truckersmp-cli; \
+		rm -f helptext; \
+	fi
```

```suggestion
	if command -v genzshcomp > /dev/null; then \
		./truckersmp-cli --help | genzshcomp > _truckersmp-cli; \
	fi
```
1. "# genzshcomp <(./truckersmp-cli --help) > _truckermp-cli" is displayed when running `make`. I don't think we need the "output".
2. Use `command -v` instead of `which`. `command` is shell built-in command that is available in `dash` and `bash` (Note: `make` uses `/bin/sh` unless `SHELL=` is specified in `Makefile`).
3. `&> /dev/null` is bash extension (try `dash -c "echo hello &> /dev/null"`). `> /dev/null` is enough.
4. Use pipe instead of creating temporary file `helptext`.

---

How did I missed that? I even tried it with a pipe…